src/Symfony/Component/Mailer/Bridge/Amazon/Transport/SesApiAsyncAwsTransport.php

```diff
@@ -107,4 +107,19 @@ private function getRecipients(Email $email, Envelope $envelope): array
             return !\in_array($address, $emailRecipients, true);
         });
     }
+
+    protected function stringifyAddresses(array $addresses): array
+    {
+        return array_map(function (Address $a) {
+            // AWS does not support UTF-8 address
+            if (preg_match('~[\x00-\x08\x10-\x19\x7F-\xFF\r\n]~', $name = $a->getName())) {
```

Comment:
What if the name includes a `"`?

`Address::toString()` currently protects from that. 

We should use `Address::getEncodedName()`  instead.

Reply:
Or... it may be okey since you base64_encode it... hm

Reply:
Exact, I tried, and the quote are base64 encoded which is fine. This is also the algorithm used in `AbstractHeader` top encode words